### DISCUSSION TITLE
Welcome to spartan Discussions!

### DISCUSSION THREAD

**Discussion by goetzrobin** [spartan-ng/spartan#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by saffabook** [comment]
Hi there, I have just become a sponsor. Love the idea. Is there a discord channel I could join to follow along, learn, contribute etc to this project maybe? Best regards.

**Reply by goetzrobin** [reply]
Thanks for your sponsorship and interest in the project! We do have a discord! Here’s the link: https://discord.gg/P7CX3fkjYD
